lib/rules/checkboxgroup.json

```diff
@@ -1,11 +1,12 @@
 {
 	"id": "checkboxgroup",
 	"selector": "input[type=checkbox][name]",
-	"tags": ["cat.forms", "best-practice"],
+	"tags": ["cat.forms", "best-practice", "deprecated"],
 	"metadata": {
 		"description": "Ensures related <input type=\"checkbox\"> elements have a group and that the group designation is consistent",
 		"help": "Checkbox inputs with the same name attribute value must be part of a group"
 	},
+	"enabled": false,
 	"all": [],
 	"any": ["group-labelledby", "fieldset"],
```

Comment:
I created an issue, because these two checks will also have to be deprecated. That has to be in a different PR though: #1733